.moban.yaml

```diff
@@ -26,7 +26,7 @@ configuration:
 targets:
   - setup.py: bears-setup.py.jj2
   - requirements.txt: requirements.txt.jj2
-  - test-requirements.txt: test-requirements.txt.jj2
+  - test-requirements.txt: core-test-requirements.txt.jj2
```

Comment:
still `core-` here ... you didnt test this locally before pushing for review.

Reply:
Sorry.